lib/order.rb

```diff
@@ -0,0 +1,20 @@
+class Order
+  attr_reader :basket, :total
+
+  def initialize
+    @basket = Hash.new(0)
+  end
+
+  def add_basket(item, qty)
+    basket[item] += quantity
+  end
+
+  def basket_summary(menu)
+    basket.map { |item, qty| "#{item} x#{qty} = £#{sprintf('%.2f', menu.dishes[item] * qty)}" }.join(', ')
```

Comment:
Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/FormatString: Favor `format` over `sprintf`. (https://github.com/bbatsov/ruby-style-guide#sprintf)